.storybook/storybook.tsx

```diff
@@ -47,7 +46,6 @@ export const StorybookUIRoot: React.FC = () => (
           />
         </Stack.Navigator>
       </NavigationContainer>
-      <ThemeSync />
```

Comment:
this is useful to manage dark/light theme switch. have you use another solution?

Reply:
ok this is what the new story is for

Reply:
It may not be a better solution, but I was experimenting with it. Perhaps we could add the toggle as part of the base ui screen.